cmd/kubeadm/app/constants/constants.go

```diff
@@ -453,7 +453,8 @@ var (
 	MinimumControlPlaneVersion = getSkewedKubernetesVersion(-1)
 
 	// MinimumKubeletVersion specifies the minimum version of kubelet which kubeadm supports
-	MinimumKubeletVersion = getSkewedKubernetesVersion(-1)
+	// Refer to https://kubernetes.io/releases/version-skew-policy/#kubelet-1
```

Comment:
Should I add some comments that kubelet version can be N-3 of the kubeadm version, as control-plane is n or n-1, this can meet N-3 of the control plane version.


```
// Allows the kubelet version to be N-3 of the kubeadm version, provided that the control plane version is either N or N-1, meeting the maximum N-3 requirement for the control plane version.
```

Reply:
actually i think we should drop this comment and just leave the old comment (line 455):
> // Refer to https://kubernetes.io/releases/version-skew-policy/#kubelet-1

the constants.go is not a place where users should read what we have. it's for us / developers to manage how kubeadm works.

if they visit the k8s.io website they can find the kubeadm and k8s skews.

Reply:
Should we update it to https://kubernetes.io/releases/version-skew-policy/?

Reply:
i think it's ok to not link to docs. these comments are for developers to understand.